gems/pending/postgres_ha_admin/postgres_ha_logger.rb

```diff
@@ -0,0 +1,23 @@
+module PostgresHaAdmin
+  class PostgresHaLogger
+    attr_reader :log_file
+
+    def initialize(log_dir)
+      @log_file = Pathname.new(log_dir).join('ha_admin.log')
```

Comment:
Sorry if this was already mentioned, is there a reason we're not creating a `@logger = Logger.new...` here instead of opening/closing the file on each `info` and `error` method call?